cycling-chd-paper/cycling-chd-paper.Rmd

```diff
@@ -116,7 +116,7 @@ A retrospective cohort study used data from a national survey of UK residents
 overweight, obesity, hypertension and diabetes in different socio-demographic
 groups. This study included public transport, walking and cycling in the
 definition of active transport and used logistic regression methods. Across all
-groups, active transport was associated with a likelihood of being overweight
+groups, active transport was associated with a (YJN edit: missing word lower??)likelihood of being overweight
```

Comment:
Yes - looks like it's missing! Please add comments like this one and fix the text directly